hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AbfsConfiguration.java

```diff
@@ -399,6 +403,10 @@ public long getAzureBlockSize() {
     return this.azureBlockSize;
   }
 
+  public boolean isCheckAccessEnabled() {
```

Comment:
Why make this optional? Can't is always be enabled?

Reply:
Currently the ABFS access() is a noop. Adding the functionality with a config flag to keep it backward compatible.